pkg/lvm/lvm_util.go

```diff
@@ -265,6 +266,38 @@ func buildLVMDestroyArgs(vol *apis.LVMVolume) []string {
 	return LVMVolArg
 }
 
+// buildLVMRestoreThinSnapshotArgs returns lvcreate command for the retore volume from thin snapshot
+func buildLVMRestoreThinSnapshotArgs(vol *apis.LVMVolume) []string {
+	var LVMRestoreThinVolArg []string
+
+	// command to create restored thin volume from thin snapshot
+	// `lvcreate -s -n <restore-volume-name>  lvmvg/<thin-snapshot-id>`
+	LVMRestoreThinVolArg = append(LVMRestoreThinVolArg, "-s", "-n", vol.Name)
+
+	if len(vol.Spec.VolGroup) != 0 {
+		LVMRestoreThinVolArg = append(LVMRestoreThinVolArg, vol.Spec.VolGroup+"/"+getLVMSnapName(vol.Spec.Source))
+	}
+
+	// -y is used to wipe the signatures before creating LVM volume
```

Comment:
What does this actually wipe?

Reply:
[fixed](https://github.com/openebs/lvm-localpv/pull/419/files/6444bacda8d8c87c2fc15b1e679e70c20550072c..d70299fe4089990f9eeb71ab682c2d622e3fbc5d)